### ISSUE TITLE
Show backLayer first

### ISSUE THREAD

**Issue by LorenzoVianello** [fluttercommunity/backdrop#80]
**Is your feature request related to a problem? Please describe.**
I'm trying to reveal backLayer first, without use it as frontLayer, in order to mantain frontLayer controllers (eg. like in spotify, where frontLayer is music bottomBar, which on click goes to music controls, that we use in frontLayer

**Describe the solution you'd like**
I'd like a solution as "toggleFirst: true/false"

**Comment by WieFel** [comment]
@LorenzoVianello ok, I think we can add a property `revealBackLayerAtStart` or similar, which defaults to `false`.
I couldn't find any other workaround.

@daadu what do you think?

**Comment by seco560** [comment]
I'd also benefit from this feature. My current workaround is to simply swap the front and back layers, but it's a bit iffy.

**Comment by daadu** [comment]
@WieFel go ahead with `revealBackLayerAtStart` idea.

**Comment by WieFel** [comment]
@daadu @LorenzoVianello @seco560
actually I **did** find a workaround just now:
```dart
class _MyAppState extends State<MyApp> with TickerProviderStateMixin {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Backdrop Demo',
      home: BackdropScaffold(
        ...
        controller: AnimationController(
            vsync: this, duration: Duration(milliseconds: 200), value: 0),
      ),
    );
  }
}
```

Use a `StatefulWidget`, use `with TickerProviderStateMixin` with your `State` and set the `BackdropScaffold` controller's initial value to `0`, so the backdrop starts in an open state.

@daadu but I think we can still add the property `revealBackLayerAtStart`, right? For better readability and easier use...

**Comment by daadu** [comment]
Yes, I forgot that custom `controller` can be passed (also we should rename to `animationController`).

`revealBackLayerAtStart` i think should still be added.

**Comment by daadu** [comment]
This feature has landed with [v0.5.2 on pub.dev](https://pub.dev/packages/backdrop/versions/0.5.2).

@LorenzoVianello Thank you for suggesting this. Please check yourself in repo README.
